### ISSUE TITLE
Video Stream Stops when Creating New Pipelines

### ISSUE THREAD

**Issue by Modelmat** [Chameleon-Vision/chameleon-vision#21]
### Steps to Reproduce
1. Create a new pipeline
2. Switch to the new pipeline
3. sometimes create a few more before it actually crashes (do not rename)

### Error
```
Exception in thread "HP Truevision HD - Vision Thread" java.lang.NullPointerException
        at com.chameleonvision.vision.pipeline.CVPipeline2d.runPipeline(CVPipeline2d.java:109)
        at com.chameleonvision.vision.pipeline.CVPipeline2d.runPipeline(CVPipeline2d.java:15)
        at com.chameleonvision.vision.VisionProcess$VisionProcessRunnable.run(VisionProcess.java:250)
        at java.base/java.lang.Thread.run(Thread.java:830)
```

### Settings File
[settings.zip](https://github.com/Chameleon-Vision/chameleon-vision/files/3934547/settings.zip)

### Other Details
I managed to create these pipelines and rename them until I made the last one where it stopped.
![image](https://user-images.githubusercontent.com/25718989/70366348-d39e4400-18ea-11ea-8c70-75240d8c2c02.png)
These files were only created:
![image](https://user-images.githubusercontent.com/25718989/70366368-e6b11400-18ea-11ea-8288-0a739e112e02.png)

Closing and restarting the server fixes the error, although I am only left with 4 pipelines ("Driver Mode" and the three visible in the directory image).

**Comment by OmerZ7** [comment]
Fixes related to #26 #27

**Comment by OmerZ7** [comment]
Related issues have been solved
Branch merge: 92ccd5aed1a815d779b8a701e8e3f56a23e01b4a
